### PR TITLE
OSSMDOC-549: Update upstream/downstream differences

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.6 - 4.11

Issue: https://issues.redhat.com/browse/OSSMDOC-549

Link to docs preview: http://file.bos.redhat.com/tokeefe/OSSMDOC-549/service_mesh/v2x/ossm-vs-community.html#ossm-vs-istio_ossm-vs-istio

Additional information:

@rcernich and @longmuir please review

Eng Review: rcernich, jwendell
QE Review: skondkar
Peer Review: jstickler

### CODE REVIEW

File: modules/ossm-vs-istio.adoc [hunk 1]
```diff
@@ -85,12 +85,16 @@ spec:
...
-
 [id="ossm-external-workloads_{context}"]
 == External workloads
 
 {SMProductName} does not support external workloads (virtual machines).
 
+[id="ossm-virtual-machine-workloads_{context}"]
+== Virtual Machine Workloads
+
+You can deploy virtual machines into OpenShift, and then integrate them into a mesh using OpenShift Virtualization.
```
Comment (jwendell) [inline modules/ossm-vs-istio.adoc:96]: This is confusing to me. The section above says we do not support it. Now we say users can deploy virtual machines. Are both statements true? Are they referring to the same or different concepts?

Comment (rh-tokeefe) [reply modules/ossm-vs-istio.adoc:96]: @jwendell These are referring to different concepts. 

Here is my take: My basic understanding is that this entry is trying to communicate that from OpenShift you can integrate VM workloads into a mesh using OpenShift Virtualization, and the prior comment is trying to communicate that you cannot have the mesh connect directly to the VM workloads.  So these are two different use cases. (Again, this my understanding from a conversation; I haven't seen this work.)

I'd like to hear your thoughts. Hopefully this is enough information to help convey the intent of what we're trying to say. If not, I can chat with Rob after he is back or possibly Jamie to see how we might be able to clarify things. I'm also open to any suggestions you might have.

**Review by rcernich — APPROVED** [review]

File: modules/ossm-vs-istio.adoc [hunk 2]
```diff
@@ -85,13 +85,17 @@ spec:
...
 [id="ossm-external-workloads_{context}"]
 == External workloads
 
-{SMProductName} does not support external workloads (virtual machines).
+{SMProductName} does not support external workloads, such as virtual machines running outside OpenShift on bare metal servers.
+
+[id="ossm-virtual-machine-support_{context}"]
+== Virtual Machine Support
+
+You can deploy virtual machines to OpenShift, and then use OpenShift Virtualization to integrate them into a mesh. You can apply a mesh policy, such as mTLS or AuthorizationPolicy, to these virtual machines, just like any other pod that is part of a mesh.
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ossm-vs-istio.adoc]: @rcernich I like the direction, and made the update with one small tweak to it:

You can deploy virtual machines to OpenShift using OpenShift Virtualization. Then, you can apply a mesh policy, such as mTLS or AuthorizationPolicy, to these virtual machines, just like any other pod that is part of a mesh.

**Comment by skondkar** [comment]
LGTM. Also verified creating the request headers example in the preview doc.

File: modules/ossm-vs-istio.adoc [hunk 3]
```diff
@@ -115,6 +119,25 @@ spec:
...
+
+[id="ossm-multicluster-configuration_{context}"]
+== Multicluster configurations
+
+{SMProductName} does not provide support for multicluster configurations.
+
+[id="ossm-certificate-signing-request_{context}"]
+== Custom Certificate Signing Requests (CSR)
+
+You cannot configure {SMProductName}  to process CSRs through the Kubernetes CA.
```
Comment (JStickler) [inline modules/ossm-vs-istio.adoc:139]: "You cannot configure {SMProductName} to process CSRs through the Kubernetes certificate authority (CA)."
Spell out the acronym for localization.

**Review by JStickler — CHANGES_REQUESTED** [review]
Just one small tweak, otherwise LGTM.  Ping me when that's updated.

File: modules/ossm-vs-istio.adoc (hunk 3 above)
Comment (rh-tokeefe) [reply modules/ossm-vs-istio.adoc]: I made the change, updated the PR, and created a new preview build.

**Review by JStickler — APPROVED** [review]
LGTM

**Pull request merged**